package.json

```diff
@@ -830,7 +830,7 @@
     "deep-freeze-strict": "^1.1.1",
     "deepmerge": "^4.2.2",
     "del": "^6.1.0",
-    "elastic-apm-http-client": "^11.0.1",
+    "elastic-apm-http-client": "^12.0.0",
```

Comment:
I wonder if this dep could just be removed. It looks to me like that refactor of synthtrace in https://github.com/elastic/kibana/pull/147036/files#diff-af5d8b1b7eea96e746bcc94f33717e59c6ef99145d2e181926cb59f353a874e4L9 removed the only usage of `elastic-apm-http-client` from Kibana.

@dgieselaar Does it sound right that synthtrace *used* to use `elastic-apm-http-client`, but stopped doing so in #147036?